tests/HooksHandlerTest.php

```diff
@@ -84,8 +92,41 @@ protected function setUp() {
 			$this->l10n,
 			$this->passwordExpiredRule,
 			$this->oldPasswordMapper,
-			$this->session
+			$this->session,
+			$this->manager,
+			$this->unConfigHandler
 		);
+
+		$this->manager->method('createNotification')
+			->will($this->returnCallback(function() {
+				$holder = [];
+				$mock = $this->createMock(INotification::class);
+				$mock->method('setApp')->will($this->returnCallback(function($app) use (&$holder, $mock) {
+					$holder['app'] = $app;
```

Comment:
there might be a better way, see https://github.com/owncloud/core/blob/master/apps/files_sharing/tests/NotifierTest.php#L145 and https://github.com/owncloud/core/blob/master/apps/files_sharing/tests/Service/NotificationPublisherTest.php#L101

I'm ok if you leave the tests as is